### PR TITLE
Add caption element

### LINKED ISSUE #115: Project: HTML best practices
All HTML examples should follow best practices.

Media
* [x] All images should include an `alt` attribute
   * [x] Create spreadsheet of missing alt attributes
* [ ] All images should aslo include a `height` and `width` (newer best practice)
* [ ] All audio files should have a transcript, captions, and description
* [ ] All video files should have a transcript
* [ ] Video and audio should have autoplay=off (not sure this is necessary anymore)

Form elements
* [x] No form control should exist without an associated label
  * [x] checkbox
  * [x] radio
  * [x] hidden
  * [ ] text
  * [x] password
  * [x] tel
  * [ ] number
  * [ ] color
  * [ ] range
  * [ ] date/time
  * [ ] select
  * [ ] textarea

* [ ] Every form control should have an `id` (except in the rare case where implicit labels are used)  
  * [x] checkbox
  * [x] radio
  * [ ] text
  * [x] password
  * [ ] tel
  * [ ] number
  * [ ] color
  * [ ] range
  * [ ] date/time
  * [ ] select
  * [ ] textarea

* [ ] All `input` elements should include a `name` (and value if needed)
  * [x] checkbox (value recommended)
  * [x] radio (value required)
  * [x] hidden (value required)
  * [ ] text
  * [x] password
  * [ ] tel
  * [ ] number
  * [ ] color
  * [ ] range
  * [ ] date/time
  * [ ] select
  * [ ] textarea

* [x] All checkboxes, radio buttons, and hidden input types should also include a `value`
   * [ ] checkboxes and radio buttons should be inside `<fieldset>`s
   * [ ] 
* [x] HTML docs should be standards compliant
  * [x] doctype
  * [x] `html` element should include a `lang` attribute.
  * [x] `charset`
  * [X] `title`
  * [X] `viewport` meta
* [x] Type attribute is NOT needed
  * [x] script
  * [x] stylesheet

Required attributes, elements, and content

* [x] Ensure `abbr` has a `title` attribute unless the abbreviation is preceded or followed by the definition.
* [ ] Ensure all links have adequate content ("click here" is a no-no).
* [ ] Every `<table>` should have a table `<caption>`
   * [ ] in code examples
   * [ ] in markdown (how is this done?)

Deprecated content

* [x] #139
* [X] Ensure `acronym` is no longer used.
* [x] Figure out if `hgroup` is back. If no, remove it from all but the essential occurrences

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I added a caption element to the Prerequisites section.
### Motivation
Adding this element will add more context to the section and make the code more accessible. 

### Related issues and pull requests
Fixes [#115](https://github.com/openwebdocs/project/issues/115)

### CODE REVIEW

File: files/en-us/learn/html/howto/add_a_hit_map_on_top_of_an_image/index.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ slug: Learn/HTML/Howto/Add_a_hit_map_on_top_of_an_image
 Here we go over how to set up an image map, and some downsides to consider first.
 
 <table>
+ <caption>Here are some things you need to know</caption>
```
Comment (dawei-wang) [inline files/en-us/learn/html/howto/add_a_hit_map_on_top_of_an_image/index.md:11]:
```suggestion
<caption>Here are some things you need to know</caption>
```

**Review by dawei-wang — COMMENTED** [review]
linting

**Comment by CBID2** [comment]
> linting

Thanks! It worked! 😄 Now, let the merging commence! 😄  👍🏾

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Comment by CBID2** [comment]
> Thanks! 🎉

You're welcome @estelle! 😊

**Pull request merged**